Framework7/src/requests/task_post.rb

```diff
@@ -0,0 +1,31 @@
+class TaskPost
+
+  def self.call_and_return_response_task(client, end_point, description)
+    url_elements = end_point
+    status_code, response_body = client.post(url_elements,{:description => description})
+    task_details = TaskDetails.new(response_body)
+    [status_code, task_details]
+  end
+  def self.call_and_return_response_task_all_details(client, end_point, description, complete , position)
```

Comment:
As we talked you should think the way to have only one method for create task operation